src/pages/signup/StepThree.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+import Button from '@mui/material/Button';
+
+
+export default function StepThree() {
+  return (
+    <div className="StepThreeForm">
+        <FormControl sx={{width: 400}} className="StepThreeInput">
+            <InputLabel>Gender</InputLabel>
+            <Select
+                labelId="demo-simple-select-label"
```

Comment:
`labelId` probably should be unique and more specific as well for these Select fields